EFDataAccessLibrary/Features/Estimates/EstimateContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace EFDataAccessLibrary.Features.Estimates
+{
+    public class EstimateContext : DbContext
+    {
+        public EstimateContext(DbContextOptions options) : base(options) { }
+        public DbSet<Estimate> Estimates { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Estimate>()

```

Comment:
Please use fluent api class insted configuring in ModelCreating if you will have 100 entities then this file will be gigant. That why we should move configuration entity to separated classes

https://www.learnentityframeworkcore.com/configuration/fluent-api